udf-compiler/src/main/scala/com/nvidia/spark/udf/GpuScalaUDF.scala

```diff
@@ -47,7 +48,7 @@ case class GpuScalaUDFLogical(udf: ScalaUDF) extends Expression {
       }
     } catch {
       case e: SparkException =>
-        System.err.println("UDF compilation failure: " + e)
+        logError("UDF compilation failure: " + e)
```

Comment:
This should be `logDebug` for now. Note that we may want to hook into what the sql plugin is doing using `spark.rapids.sql.explain` later, where if you pick `spark.rapids.sql.explain=ALL` for example, we could turn this into a print just for the case where the user wants to know what happened.

Reply:
Updated.